sdk/Cargo.toml

```diff
@@ -47,8 +35,8 @@ serialize_thumbnails = []
 no_interleaved_io = ["file_io"]
 fetch_remote_manifests = ["dep:wasi"]
 json_schema = ["dep:schemars"]
-openssl = ["dep:openssl"]
 pdf = ["dep:lopdf"]
```

Comment:
Why move this? These were in (roughly) alphabetical order before

Reply:
Artifact from changing things around. I fixed this in latest commit, but I think we should add a toml formatter in here somewhere.

Reply:
Once upon a time, I was contributing to such a tool: https://github.com/FuelLabs/cargo-toml-lint

Looks like it's become abandon-ware. :-(

Reply:
Here's what I use: https://github.com/tamasfe/taplo